tools/db/migrate-sso-feature-flag/src/Work.hs

```diff
@@ -0,0 +1,56 @@
+{-# LANGUAGE OverloadedStrings          #-}
+{-# LANGUAGE ScopedTypeVariables        #-}
+{-# LANGUAGE StandaloneDeriving         #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE LambdaCase                 #-}
+
+{-# OPTIONS_GHC -Wno-orphans -Wno-unused-imports #-}
+
+module Work where
+
+import Imports
+
+import Brig.Types hiding (Client)
+import Cassandra
+import Data.Conduit
+import Data.Conduit.Internal (zipSources)
+import Data.Id
+import Data.Misc
+import Galley.Types.Teams.SSO
+import Galley.Data.Instances ()
+import System.Logger (Logger)
+import UnliftIO.Async (pooledMapConcurrentlyN)
+
+import qualified Data.Conduit.List as C
+import qualified System.Logger as Log
+
+deriving instance Cql Name
+
+runCommand :: Logger -> ClientState -> ClientState -> IO ()
+runCommand l spar galley = do
+    runConduit
+        $ zipSources (C.sourceList [(1::Int32) ..])
```

Comment:
Fancy! :)

Reply:
We just stole it from @neongreen  :-)

Reply:
I can't even recall writing it 🤔At any rate, you're welcome!